Recast/PodcastDetailViewController.swift

```diff
@@ -190,6 +190,15 @@ extension PodcastDetailViewController: UITableViewDataSource {
 // MARK: - episodeTableView Delegate
 extension PodcastDetailViewController: UITableViewDelegate {
 
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
```

Comment:
are we going to change this to launch the player on tap of the play button so that other interactions on the cell are possible?

Reply:
Yeah I did this because I wasn't sure if we decided on what other actions happen when you tap on the cell.